cmd/legacy/heartbeat/params.go

```diff
@@ -622,21 +622,24 @@ func loadProjectParams(v *viper.Viper) (ProjectParams, error) {
 func parseBoolOrRegexList(s string) ([]regex.Regex, error) {
 	var patterns []regex.Regex
 
+	s = strings.Trim(s, "\n\r\t ")
+
 	switch {
-	case strings.Trim(s, "\n\r\t ") == "":
+	case s == "":
 		break
-	case strings.ToLower(strings.Trim(s, "\n\r\t ")) == "false":
+	case strings.ToLower(s) == "false":
 		break
-	case strings.ToLower(strings.Trim(s, "\n\r\t ")) == "true":
+	case strings.ToLower(s) == "true":
 		patterns = []regex.Regex{matchAllRegex}
 	default:
 		splitted := strings.Split(s, "\n")
```

Comment:
will this split work since you already trimmed by `\n\r\t ` before?

Reply:
Yes, before I only removed whitespace from start and end so it can still split by whitespace contained within the string.

Reply:
I would consider having tests for it.

Reply:
Added a test for it with b71f5c1d521ed994b65b7610efaa9d62d1282ed7.